tests/build/jvm-build.go

```diff
@@ -342,10 +376,20 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", Label("jv
 					}
 				}
 				if abComplete && dbComplete {
-					return true
+					return true, nil
 				}
-				return false
-			}, 2*timeout, interval).Should(BeTrue(), "timed out waiting for some artifactbuilds/dependencybuilds to complete")
+				return false, nil
+			})
+			if err != nil {
+				ciRepoName := os.Getenv("REPO_NAME")
+				// Fail only in case the test was run from jvm-build-service repo or locally
+				if ciRepoName == "jvm-build-service" || ciRepoName == "" {
+					Fail("timed out waiting for some artifactbuilds/dependencybuilds to complete")
+				} else {
+					doCollectLogs = true
+					Skip("SKIPPING: unstable feature: timed-out when waiting for some artifactbuilds and dependencybuilds complete")
```

Comment:
awesome @psturc 

yeah if this works as expected this way we can see if those gradle 404's persist next week and Stuart / David can take a look in jvm-build-service PRs